build/cli.js

```diff
@@ -410,6 +415,13 @@ program
         return;
       }
 
+      if (options.unslug && CONTENT_TRANSLATED_ROOT) {
+        const documentsMoved = unslugAllLocales();
+        if (documentsMoved !== 0) {
+          console.log("Please unslug the translated content repo");
```

Comment:
What does that mean? I already used `--unslug` so why is it saying this to me?

Reply:
`--unslug` means: Please run unslug before building, moving all things into the right place. Hopefully nothing had be moved because everything is up to date. So print something if that's not the case.

This option will go away and always be true when we're all unslugged. I added this to allow this PR to be merged and automatically unslug everything.